### DISCUSSION TITLE
What astrological symbol does the crab represent?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#392] [Galaxy Trivia] [answered]
**Hint:** 🦀♋

**Comment by peterbaccaro** [comment]
**Cancer** is the astrological symbol that is represented by a crab.

**Comment by thuongtruong1009** [comment, marked as answer]
The astrological symbol that the crab represents is the zodiac sign of Cancer. The crab is often used as a symbol for Cancer because, like a crab, people born under this sign are believed to be protective and defensive of their home and loved ones. Additionally, the hard shell of a crab is also seen as a symbol for the tough exterior that many Cancer individuals have, but which hides a softer, more sensitive side underneath.
